python/rikai/types/vision.py

```diff
@@ -93,13 +102,18 @@ def from_pil(img: PILImage, uri: Union[str, Path]) -> Image:
             An PIL Image instance
         uri : str or Path
             The URI to store the image externally.
+        format : str, optional
+            The image format to save as. See
+            `supported formats <https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.save>`_ for details.
```

Comment:
@eddyxu do you see a difference between this line and line 78 above? For some reason the linter likes that line but doesn't like this one...